CMakeLists.txt

```diff
@@ -93,43 +93,38 @@ else()
   if(WITH_SNAPPY)
     find_package(snappy REQUIRED)
     add_definitions(-DSNAPPY)
-    include_directories(${SNAPPY_INCLUDE_DIR})
```

Comment:
@tchaikov Why do you remove this line? Seems that cannot find it in include path. Errors like 
`/root/cs/suzanwen/repos/C++/storage/rocksdb_xxx/rocksdb/util/compression.h:31:20: fatal error: snappy.h: No such file or directory
 #include <snappy.h>
                    ^
compilation terminated.`

Reply:
because, IIUC, snappy::snappy should be able to populate the "INSTALL_INTERFACE" property to the library/executable linked against it, so cmake will determine the correct build property of consumer of snappy::snappy and library publicly linked against this library.

https://github.com/facebook/rocksdb/blob/f65ec09ef86c8dffd20aa63a12126a23d6d3c5b4/cmake/modules/Findsnappy.cmake#L28

Reply:
Yeah, that's right for the most targets, such as `ROCKSDB_STATIC_LIB`.  But not for `TESTUTILLIB`. Plz have a glance at the following snippet.
https://github.com/facebook/rocksdb/blob/f65ec09ef86c8dffd20aa63a12126a23d6d3c5b4/CMakeLists.txt#L1072

`snappy.h` was included in `db/db_test_util.cc`(just as the source file in target `TESTUTILLIB`). Porbably we need make it works fine with `target_link_libraries(${TESTUTILLIB} ${LIBS})`.

Reply:
@tchaikov https://github.com/facebook/rocksdb/pull/6043 pr as it is, plz make a review.